### PR TITLE
#183031599: ENABLE USERS TO VIEW AND EDIT THEIR PROFILES

### PR DESCRIPTION
**Pull request by narravabrion**
#### What does this Pull Request do?
+ Enable users to view and edit their profiles.
#### Description of Task to be completed?
+ Users should be able to ammend/edit their profiles depending on change of preference to the full disclosure of the individual user
#### How should this be manually tested?
+ from the root of the project run `npm run start`
+ from the `navbar` select `profile`
+ edit profile if you have the write permission
#### Any background context you want to provide?
+ Users should have access to their profile.
+ Users should be able to edit their profile to their preference.
#### What are the relevant pivotal tracker story?
[pivotal card](https://www.pivotaltracker.com/n/projects/2570604/stories/183031599)
#### Screenshots (if appropriate)
NA
#### Questions:
NA

### CODE REVIEW

File: src/pages/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { AiFillEdit, AiFillDelete } from 'react-icons/ai';
+import { Link } from 'react-router-dom';
+// import { Link, useLocation, useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { privateLinks } from '../constants/links';
+import useAuth from '../hooks/useAuth';
+// import axios from '../api/axios';
+// import useRefreshToken from '../hooks/useRefreshToken';
```
Comment (KaigaMunene) [inline src/pages/Profile.jsx]: remove unnecessary comments

File: src/pages/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import useAxiosPrivate from '../hooks/useAxiosPrivate';
+
+function Profile() {
+  const axiosPrivate = useAxiosPrivate();
+  // const navigate = useNavigate();
+  //   const refresh = useRefreshToken();
+  // const location = useLocation();
```
Comment (KaigaMunene) [inline src/pages/Profile.jsx]: same for here

File: src/pages/Profile.jsx (hunk 1 above)
Comment (narravabrion) [reply src/pages/Profile.jsx]: done

File: src/pages/Profile.jsx (hunk 2 above)
Comment (narravabrion) [reply src/pages/Profile.jsx]: done

File: src/pages/Profile.jsx (hunk 1 above)
Comment (KaigaMunene) [inline src/pages/Profile.jsx]: what about this

File: src/pages/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { AiFillEdit, AiFillDelete } from 'react-icons/ai';
+import { Link } from 'react-router-dom';
+// import { Link, useLocation, useNavigate } from 'react-router-dom';
```
Comment (KaigaMunene) [inline src/pages/Profile.jsx]: and this ??

Comment (narravabrion) [reply src/pages/Profile.jsx]: They're not in my local repo

**Pull request merged**